### DISCUSSION TITLE
Reliable Update Method

### DISCUSSION THREAD

**Discussion by matteoveglia** [horilla-opensource/horilla#20] [Q&A]
Hi,

I'm super keen to try out Horilla, I've been searching for something like this for a while and what has been put together here looks great!

Question though, as I don't see a docker image available (which would be a great addition), is there a reliable way to keep my instance updated to the latest codebase?

I'm assuming I should just reclone the repo, build and overwrite each time?

**Comment by horilla-opensource** [comment]
Hi @matteoveglia ,
Happy to hear that you are interested in our system.
Currently as we our under our development phase and yet to release our first stable version, we are not able to currently provide you with a stable docker image of the system.
Soon we will be releasing our first stable version, followwd by its docker image.
We'll keep you updated of the same.
